apps/indexer/ponder.schema.ts

```diff
@@ -83,6 +83,8 @@ export const votingPowerHistory = onchainTable(
     votingPower: drizzle.bigint("voting_power").notNull(),
     delta: drizzle.bigint("delta").notNull(),
     timestamp: drizzle.bigint().notNull(),
+    // the voting power change non-negative to correlate with the transfer and delegation
+    deltaMod: drizzle.bigint("delta_mod").notNull(),
```

Comment:
Primary key constraint violation: The `transfer` table primary key includes `transactionHash`, but `transactionHash` is nullable. Primary key columns cannot be null, which will cause database insertion failures.